scss/_patterns_chip.scss

```diff
@@ -1,18 +1,19 @@
 @import 'settings';
 
 @mixin vf-p-chip {
-  $chip-vertical-padding: map-get($nudges, nudge--small);
-  $chip-height: map-get($line-heights, small) + 2 * $chip-vertical-padding;
-  $chip-margin-bottom: $spv-outer--small-scaleable - 2 * $chip-vertical-padding;
+  $chip-line-height: map-get($line-heights, default-text);
+  $chip-vertical-padding: 0;
+  $chip-height: $chip-line-height + 2 * $chip-vertical-padding;
+  $chip-margin-top: map-get($nudges, nudge--p) - $chip-vertical-padding;
 
   .p-chip {
     @extend %small-text;
 
     background-color: rgba($color-x-dark, 0.05);
     border-radius: $chip-height * 0.5;
-    display: inline;
-    margin-bottom: $chip-margin-bottom;
-    margin-right: $sph-inner--small;
+    display: inline-block;
+    line-height: $chip-line-height;
+    margin: $chip-margin-top $sph-inner--small 0 0;
     padding: $chip-vertical-padding $sph-inner;
```

Comment:
I would suggest changing this from a result of:
```css
padding: 0 1rem;
```
to
```scss
padding-left: $sph-inner;
padding-right: $sph-inner;
```
Long-form is generally preferred over shorthand as its more specific and results it less complicated overrides.